Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/problem-groups/problem-group-form/ProblemGroupForm.tsx

```diff
@@ -147,16 +147,6 @@ const ProblemGroupForm = (props: IProblemFormProps) => {
                 <Typography variant="h4" className="centralize">
                     Problem Group Administration Form
                 </Typography>
-                <FormControl className={formStyles.inputRow}>
-                    <TextField
-                      variant="standard"
-                      label={ID}
-                      value={currentProblemGroup?.id}
-                      InputLabelProps={{ shrink: true }}
```

Comment:
If you remove the input entirely, will not this create a bug in the back end? When ProblemGroupId is sent as null, how the back end will know which entity is being updated?